components/suggestion/suggestion_list.jsx

```diff
@@ -224,18 +225,17 @@ export default class SuggestionList extends React.PureComponent {
 
         const contentStyle = {maxHeight};
 
-        return (
-            <div className={mainClass}>
-                <div
-                    id='suggestionList'
-                    ref={this.contentRef}
-                    style={{...contentStyle}}
-                    className={contentClass}
-                    onMouseDown={this.props.preventClose}
-                >
-                    {items}
-                </div>
+        return this.props.supressLoadingSpinner ? '' : (<div className={mainClass}>
```

Comment:
How does it look without suppressing the spinner?

Reply:
It renders an empty line and/or a spinner even without any results because its mounting this component.

Reply:
Gotcha, it doesn't work even if the spinner is hidden? wondering if we can just not include the spinner in the `items` (earlier in the function) rather than hiding this whole component. If it doesn't work then all good, just a thought.

Reply:
I tried that at first and it didn't hide the line 🙃

Reply:
oh okay, not to thrash on the change but then shouldn't this be more like `hideSuggestionList` or something similar? and I imagine returning `null` here doesn't work either?